app/controllers/status_updates_controller.rb

```diff
@@ -0,0 +1,14 @@
+class StatusUpdatesController < ApplicationController
+  wrap_parameters false
+
+  def create
```

Comment:
Do we need to think about how we can secure this webhook now or is that for later? Because if the route is public, it does mean anyone can change the status of delivery attempts right now.

Reply:
Yep, we're proposing we add a HTTP header that sets a bearer token.

I held off implementing this as we're yet to confirm with Notify.

(more context: https://docs.google.com/a/digital.cabinet-office.gov.uk/document/d/1g7NlOQT-e02n1nL4DXSNfrvZfFmkQtiAmh5IjNx-10A/edit?usp=sharing)